realm/realm-library/src/androidTest/java/io/realm/LinkingObjectsUnmanagedTests.java

```diff
@@ -53,7 +53,7 @@ public void tearDown() {
         }
     }
 
-    // When unmanaged, an object's backlinks fields have their initialized value (probably null).
```

Comment:
Well as far as I know you can't really obtain an "empty RealmResults", and that's the only other thing that would make sense for it to be.